### PR TITLE
new SOBJ serializer

### PR DESCRIPTION
**Pull request by rayatus**
new SOBJ serializer relying on GENERIC serializer (#4835)

### CODE REVIEW

File: src/objects/zcl_abapgit_object_sobj.clas.abap [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        io_field_rules = get_field_rules( )
+        is_item        = ms_item
+        iv_language    = mv_language.
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~changed_by.
+
+    SELECT SINGLE crea_user, chan_user INTO @DATA(ls_userinfo) FROM tojtb WHERE name = @ms_item-obj_name.
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_sobj.clas.abap:43]: downport to 702,

```suggestion
    SELECT SINGLE crea_user, chan_user INTO ls_userinfo FROM tojtb WHERE name = ms_item-obj_name.
```

**Comment by mbtools** [comment]
nice. please add a test object to https://github.com/abapGit-tests/SOBJ

**Comment by rayatus** [comment]
> nice. please add a test object to https://github.com/abapGit-tests/SOBJ

@mbtools there is [PR ](https://github.com/abapGit-tests/SOBJ/pull/1) waiting for it

**Comment by mbtools** [comment]
thanks. i will test it next week

**Comment by rayatus** [comment]
@mbtools , just wondering if you had the time to have a look at this PR.
Let me know if I could help someway or if there is anything pending from my side. ;)

File: src/objects/zcl_abapgit_object_sobj.clas.abap [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    )->add(
+      iv_table     = 'TOJTB'
+      iv_field     = 'REL_DATE'
+      iv_fill_rule = zif_abapgit_field_rules=>c_fill_rule-date
+    )->add(
+      iv_table     = 'TOJTB'
+      iv_field     = 'REL_TIME'
+      iv_fill_rule = zif_abapgit_field_rules=>c_fill_rule-time ).
+
+*    ri_rules->add(
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_sobj.clas.abap:175]: remove commented code

File: src/objects/zcl_abapgit_object_sobj.clas.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <VSEOCLASS>
+    <CLSNAME>ZCL_ABAPGIT_OBJECT_SOBJ</CLSNAME>
+    <LANGU>E</LANGU>
+    <DESCRIPT>SOBJ - Business Object</DESCRIPT>
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_sobj.clas.xml:8]: change the description to "SOBJ - Business Object Type"

**Review by mbtools — CHANGES_REQUESTED** [review]
Sorry for the delay. Tests with the example repo worked fine. Just a couple small changes

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**